src/lib/MTL1.hs

```diff
@@ -290,6 +290,13 @@ runFallibleT1 m =
     Left errs -> return $ Failure errs
 {-# INLINE runFallibleT1 #-}
 
+ignoreFallibleT1 :: Monad1 m => FallibleT1 m n a -> m n a
+ignoreFallibleT1 m =
```

Comment:
Isn't this just `liftM ignoreExcept $ runFallibleT1`? I don't think we need a helper for that

Reply:
Sure. I'll delete.